java/flight/flight-sql/src/main/java/org/apache/arrow/flight/sql/FlightSqlClient.java

```diff
@@ -77,6 +77,7 @@
 /**
  * Flight client with Flight SQL semantics.
  */
+@SuppressWarnings({"ProtoBuilderReturnValueIgnored", "ReturnValueIgnored"})
```

Comment:
Hmm, this is a very wide scope to suppress. Is it possible to refactor things?

Reply:
Changed